plugins/kubernetes/api-report.md

```diff
@@ -108,6 +108,38 @@ export interface DeploymentResources {
   replicaSets: V1ReplicaSet[];
 }
 
+// Warning: (ae-missing-release-tag) "DetectedError" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```

Comment:
Let's not introduce any new API report warnings, please handle these

Reply:
Sure :+1: I'll fix this once there's an agreement if this will go ahead as is or not.